lib/resources/r4/encounter_patch.yaml

```diff
@@ -64,3 +64,36 @@ operations:
         <li>`Encounter.type` value should be a list.</li>
         <li>When a list is provided, the first value would be used to patch `Encounter.type`.</li>
       </ul>
+
+  - name: replace-extension
+    path: /extension
+    operation: replace
+    type: List of Extension
+    url: http://hl7.org/fhir/encounter-definitions.html#Encounter.extension
+    description: The aray of extension of encounter.
+    example: |
+      {
+        "op": "replace",
+        "path": "/extension",
+        "value":  [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status",
+            "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003489",
+                "code": "40016441",
+                "display": "1st Attempt - Left Message",
+                "userSelected": true
+              }
+            ],
+            "text": "1st Attempt - Left Message"
+           }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>`Encounter.extension` value should be a list.</li>
+        <li>When a list is provided, the first value would be used to patch `Encounter.extension`.</li>
```

Comment:
If you have a list of two supported extensions to patch (which will be possible soon), this makes it sound like only one will be updated

Reply:
Now, since the second extension is not patched yet,  will it sound good , if we say otherwise.
I guess it could be changed during the time when second extension patch is worked on.

Reply:
I guess im thinking that because we validate single occurrence i dont know if this note is needed...?

Reply:
We should remove this and instead have a list of supported extensions.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/a9a65e45d04546e76fa09cc9d8b4ba7395d2f7d2#diff-97d84edc87e609cdbf80d6ad20cbc681R98

Reply:
Changed the example codevalue to point to the C1941 domain:

https://github.com/cerner/fhir.cerner.com/commit/9a21168c32e534377b2b547bec16ba3176de5ff8#diff-8915968a455f59526d7e4635365d9792R1121

https://github.com/cerner/fhir.cerner.com/commit/9a21168c32e534377b2b547bec16ba3176de5ff8#diff-97d84edc87e609cdbf80d6ad20cbc681R85